<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-SOD77OH" name="WebDriverTest.RecordTestingForSingleRecords" tests="8" failures="0" timestamp="17 May 2017 07:29:11 GMT" time="1412.080" errors="3">
  <testcase name="refreshOperation" time="162.812" classname="WebDriverTest.RecordTestingForSingleRecords"/>
  <testcase name="deleteOperation" time="52.958" classname="WebDriverTest.RecordTestingForSingleRecords"/>
  <testcase name="insertOperation" time="190.655" classname="WebDriverTest.RecordTestingForSingleRecords"/>
  <testcase name="updateOperation" time="275.925" classname="WebDriverTest.RecordTestingForSingleRecords">
    <error type="java.lang.AssertionError" message="Errorjava.lang.AssertionError: 96199[.addressbookList.addressbook.addressbookAddress.addr1=address one]..addressbookList.addressbook.addressbookAddress.country
Expected: United States
     got: 
 ; 96199[.addressbookList.addressbook.addressbookAddress.addr1=address one]..custentity_free_from_text
Expected: testing fft
     got: 
 ; 96199[.addressbookList.addressbook.addressbookAddress.addr1=address one]..custentity_pick_list
Expected: Organization Expense
     got: 
 ; 96199[.addressbookList.addressbook.addressbookAddress.addr1=address one]..email
Expected: jacksom.james@celigo.com
     got: 
">
      <![CDATA[java.lang.AssertionError: Errorjava.lang.AssertionError: 96199[.addressbookList.addressbook.addressbookAddress.addr1=address one]..addressbookList.addressbook.addressbookAddress.country
Expected: United States
     got: 
 ; 96199[.addressbookList.addressbook.addressbookAddress.addr1=address one]..custentity_free_from_text
Expected: testing fft
     got: 
 ; 96199[.addressbookList.addressbook.addressbookAddress.addr1=address one]..custentity_pick_list
Expected: Organization Expense
     got: 
 ; 96199[.addressbookList.addressbook.addressbookAddress.addr1=address one]..email
Expected: jacksom.james@celigo.com
     got: 

	at org.testng.Assert.fail(Assert.java:94)
	at util.CommonLibrary.compareData(CommonLibrary.java:989)
	at WebDriverTest.RecordTestingForSingleRecords.getFromNsAndCompare(RecordTestingForSingleRecords.java:199)
	at WebDriverTest.RecordTestingForSingleRecords.updateOperation(RecordTestingForSingleRecords.java:269)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </error>
  </testcase> <!-- updateOperation -->
  <testcase name="insertOperation" time="252.101" classname="WebDriverTest.RecordTestingForSingleRecords"/>
  <testcase name="deleteOperation" time="66.669" classname="WebDriverTest.RecordTestingForSingleRecords"/>
  <testcase name="refreshOperation" time="199.830" classname="WebDriverTest.RecordTestingForSingleRecords">
    <error type="java.lang.AssertionError" message="Errorjava.lang.AssertionError: 96199[.addressbookList.addressbook.addressbookAddress.addr1=address one]..addressbookList.addressbook.addressbookAddress.country
Expected: United States
     got: 
">
      <![CDATA[java.lang.AssertionError: Errorjava.lang.AssertionError: 96199[.addressbookList.addressbook.addressbookAddress.addr1=address one]..addressbookList.addressbook.addressbookAddress.country
Expected: United States
     got: 

	at org.testng.Assert.fail(Assert.java:94)
	at util.CommonLibrary.compareData(CommonLibrary.java:989)
	at WebDriverTest.RecordTestingForSingleRecords.getFromNsAndCompare(RecordTestingForSingleRecords.java:199)
	at WebDriverTest.RecordTestingForSingleRecords.refreshOperation(RecordTestingForSingleRecords.java:305)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </error>
  </testcase> <!-- refreshOperation -->
  <testcase name="updateOperation" time="211.130" classname="WebDriverTest.RecordTestingForSingleRecords">
    <error type="java.lang.AssertionError" message="Errorjava.lang.AssertionError: 96099[.category=Customer7]..email
Expected: started@customer1.com
     got: 
">
      <![CDATA[java.lang.AssertionError: Errorjava.lang.AssertionError: 96099[.category=Customer7]..email
Expected: started@customer1.com
     got: 

	at org.testng.Assert.fail(Assert.java:94)
	at util.CommonLibrary.compareData(CommonLibrary.java:989)
	at WebDriverTest.RecordTestingForSingleRecords.getFromNsAndCompare(RecordTestingForSingleRecords.java:199)
	at WebDriverTest.RecordTestingForSingleRecords.updateOperation(RecordTestingForSingleRecords.java:269)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </error>
  </testcase> <!-- updateOperation -->
</testsuite> <!-- WebDriverTest.RecordTestingForSingleRecords -->
